GitUI/GitUIExtensions.cs

```diff
@@ -56,14 +59,19 @@ public static class GitUIExtensions
             if (item.Item.IsNew || firstId is null || (!item.Item.IsDeleted && FileHelper.IsImage(item.Item.Name)))
             {
                 // View blob guid from revision, or file for worktree
-                return fileViewer.ViewGitItemRevisionAsync(item.Item, item.SecondRevision.ObjectId, openWithDiffTool);
+                await fileViewer.ViewGitItemRevisionAsync(item.Item, item.SecondRevision.ObjectId, openWithDiffTool);
+                return;
             }
 
             if (item.Item.IsRangeDiff)
             {
-                // This command may take time, give an indication of what is going on
-                // The sha are incorrect if baseA/baseB is set, to simplify the presentation
-                fileViewer.ViewText("range-diff.sh", $"git range-diff {firstId}...{item.SecondRevision.ObjectId}");
+                // Git range-diff has cubic runtime complexity and can be slow and memory consuming,
+                // give an indication of what is going on
+                var range = item.BaseA is null || item.BaseB is null
+                ? $"{firstId}...{item.SecondRevision.ObjectId}"
+                : $"{item.BaseA}..{firstId} {item.BaseB}..{item.SecondRevision.ObjectId}";
```

Comment:
Indent
```suggestion
                    ? $"{firstId}...{item.SecondRevision.ObjectId}"
                    : $"{item.BaseA}..{firstId} {item.BaseB}..{item.SecondRevision.ObjectId}";
```
?

Reply:
I expect auto indent should handle this, disappointed every time it is not.